internal/cmd/backup.go

```diff
@@ -87,6 +87,9 @@ func registerBackupCmd(rootCmd *cobra.Command) {
 	backupCmd.AddCommand(backupRestoreCmd)
 	backupRestoreCmd.Flags().Int("batch-size", 1_000, "restore relationship write batch size")
 	backupRestoreCmd.Flags().Int64("batches-per-transaction", 10, "number of batches per transaction")
+	backupRestoreCmd.Flags().Bool("skip-conflicts", false, "skips any batch found to be conflicting")
```

Comment:
are `skip-conflicts` and `touch-conflicts` exclusive? If so, maybe we have a combined flag with an enum option?

Reply:
yeah Jimmy shared recently https://github.com/thediveo/enumflag - I'll look into it

Reply:
yeah it's a bit of a shame because i think it only works with flags registered with `Var()`.

either way we should make this something like `--on-conflict error/skip/touch`

Reply:
Addressed. Ended up creating a `Must*` style of helper we could add to `cobrautil`